src/server/docker.rs

```diff
@@ -791,6 +795,11 @@ impl<'os, O: CurrentOs + ?Sized> DockerMethod<'os, O> {
                 DROP ROLE {};
             "###, role=tmp_role)).await?;
 
+            // TODO: skip if --non-interactive
+            credentials::ask_update_credentials(
```

Comment:
I don't think we need to ask here. We already rewrite credentials to add a certificate, just fix the credentials anyways. I think it's fine (to not ask) even if we keeping certificate the same, as the expectation is the same: your credentials might be updated on server upgrade.